lib/Parse/ParseStmt.cpp

```diff
@@ -1846,8 +1907,8 @@ Parser::parseStmtConditionElement(SmallVectorImpl<StmtConditionElement> &result,
 /// The use of expr-basic here disallows trailing closures, which are
 /// problematic given the curly braces around the if/while body.
 ///
-ParserStatus Parser::parseStmtCondition(StmtCondition &Condition,
-                                        Diag<> DefaultID, StmtKind ParentKind) {
+ParserStatus Parser::parseStmtCondition(StmtCondition &Condition, Diag<> DefaultID, StmtKind ParentKind) {
```

Comment:
I would say same thing here about the code formatting, I think the way it was formatted previously was intentional and not something we need to change.

Reply:
This means that the code red is good. There are no errors